### PR TITLE
Added tabs feature

### PR DESCRIPTION
**Pull request by afzalsayed96**
## Brief description

Tabs using CSS

## Developer Certificate of Origin

- [x] I certify that these changes according to the Developer Certificate of Origin 1.1 as described at <https://developercertificate.org/>.

## Sample pictures

demo

## Further details

Inspired by: https://codepen.io/oknoblich/pen/tfjFl

### CODE REVIEW

**Comment by rhyneav** [comment]
Awesome! Love that tabs can be done only using CSS. Could you please resolve the conflicts that popped up when I merged in another feature? I'd be more than happy to merge this in afterward.

**Comment by afzalsayed96** [comment]
Will need write access to resolve conflicts I guess

File: src/tabs.less [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        color: @muted;
+        cursor: pointer;
+    }
+
+    input:checked+label {
+        color: @primary;
+        border-bottom: solid 3px @secondary;
+    }
+
+    #tab1:checked~#content1,
```
Comment (Fraham) [inline src/tabs.less]: should this css be here, it seems like this is very much linked to your example and wouldn't be reusable for someone using the framework

Comment (afzalsayed96) [reply src/tabs.less]: Thanks for pointing out

Comment (afzalsayed96) [reply src/tabs.less]: @Fraham I've added loop to select tabs id from 1 to 5. Can you help me get this id dynamically from HTML or should we arbitrarily set this limit up 10 or 20 tabs?

File: src/tabs.less [hunk 2]
```diff
@@ -27,11 +27,12 @@
         border-bottom: solid 3px @secondary;
     }
 
-    #tab1:checked~#content1,
-    #tab2:checked~#content2,
-    #tab3:checked~#content3,
-    #tab4:checked~#content4 {
-        display: block;
+    .loop(@num) when (@num > 0) {
```
Comment (TotomInc) [inline src/tabs.less]: Looks really great!

**Comment by afzalsayed96** [comment]
ping @rhyneav :)

**Comment by rhyneav** [comment]
Looks great! Thank you for this!

**Pull request merged**